src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private double firstLeg;
+    private double secondLeg;
+
+    public Rectangle(String color, double firstLeg, double secondLeg) {
```

Comment:
Passing random values to the constructor is a good practice, but make sure that these values are not generated within the figure classes themselves.